notebook_analyzer/connector/get_data.py

```diff
@@ -173,9 +193,9 @@ def cells(self):
         return self._cells
 
     def get_script_source(self):
-
-        route = os.path.abspath('/Users/konstantingrotov/Documents/datasets/20kk_dataset') + '/'
-        path = route + self._metadata['name']
+        #route = os.path.abspath('/home/ubuntu/jupyter-corpus/jpt-nb-corpus/ databases/20kk_dataset') + '/'
```

Comment:
path should be an argument to a function

Reply:
`os.path.abspath(self._metadata['name'])` is an absolute path with name of script. I decided to do it for more convenience in processing scripts, without binding to separate directories

Reply:
This comment is no longer relevant to the code